static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -255,6 +263,7 @@ const buildPageTargetting = (
             // and can be decomissioned after Pascal and D&I no longer need the flag.
             inskin: inskinTargetting(),
             urlkw: getUrlKeywords(page.pageId),
+            ccpa: getCcpaValue(ccpaState),
```

Comment:
Oops I didn't see that, @ripecosta  but the name of the flag should be 'rdp' according to https://trello.com/c/W1qrG2tq/847-add-the-ccpa-consent-state-to-dfp-targeting-key-values

Reply:
I discussed this with D&I and the outcome was to name it 'ccpa'. Is there a dependency here I'm not aware of or are we ok to go with this decision?